app/presenters/licences/view-licence.presenter.js

```diff
@@ -180,6 +180,9 @@ function _parseAbstractionsAndSourceOfSupply (permitLicence) {
 
   const abstractionPointsCaption = uniqueAbstractionPoints.length > 1 ? 'Points of abstraction' : 'Point of abstraction'
 
+  console.log(abstractionPoints)
+  console.log(uniqueAbstractionPoints)
```

Comment:
If we do need to log stuff we have a different function for that. But I'm guessing these are just left over from debugging! 🐛🐞

Reply:
ah sorry about that! 😅